Cargo.toml

```diff
@@ -112,7 +112,7 @@ futures = "0.3"
 futures-core = "0.3.31"
 futures-util = "0.3.31"
 git-version = "0.3.9"
-golem-wit = { version = "=1.1.2" }
+golem-wit = { git = "https://github.com/golemcloud/golem-wit.git", branch = "add_fork" }
```

Comment:
Don't forget to change this before merging

Reply:
No I won't. I will be doing this before merge.